dali/test/python/test_pipeline.py

```diff
@@ -600,6 +600,68 @@ def define_graph(self):
+def test_equal_nvJPEGDecoderCrop_nvJPEGDecoder():
+    """
+        Comparing results of pipeline: (nvJPEGDecoder -> Crop), with the same operation performed by fused operator
+    """
+    batch_size =128
+
+    class SequencialOPPipeline(Pipeline):
+        def __init__(self, batch_size, num_threads, device_id, num_gpus):
+            super(SequencialOPPipeline, self).__init__(batch_size,
+                                             num_threads,
+                                             device_id)
+            self.input = ops.CaffeReader(path = caffe_db_folder, shard_id = device_id, num_shards = num_gpus)
+            self.decode = ops.nvJPEGDecoder(device = "mixed", output_type = types.RGB)
+            self.pos_rng_x = ops.Uniform(range = (0.0, 1.0), seed=1234)
+            self.pos_rng_y = ops.Uniform(range = (0.0, 1.0), seed=5678)
+            self.crop = ops.Crop(device="gpu", crop =(224,224))
+
+        def define_graph(self):
+            self.jpegs, self.labels = self.input()
+
+            pos_x = self.pos_rng_x()
+            pos_y = self.pos_rng_y()
+            images = self.decode(self.jpegs)
+            crop = self.crop(images, crop_pos_x=pos_x, crop_pos_y=pos_y)
+            return (crop, self.labels)
+
+
+        def iter_setup(self):
+            pass
+
+    class FusedOPPipeline(Pipeline):
+        def __init__(self, batch_size, num_threads, device_id, num_gpus):
+            super(FusedOPPipeline, self).__init__(batch_size,
+                                             num_threads,
+                                             device_id)
+            self.input = ops.CaffeReader(path = caffe_db_folder, shard_id = device_id, num_shards = num_gpus)
+            self.pos_rng_x = ops.Uniform(range = (0.0, 1.0), seed=1234)
+            self.pos_rng_y = ops.Uniform(range = (0.0, 1.0), seed=5678)
+            self.decode = ops.nvJPEGDecoderCrop(device = 'mixed', output_type = types.RGB, crop = (224, 224))
+
+        def define_graph(self):
+            self.jpegs, self.labels = self.input()
+            pos_x = self.pos_rng_x()
+            pos_y = self.pos_rng_y()
+            images = self.decode(self.jpegs, crop_pos_x=pos_x, crop_pos_y=pos_y)
+            return (images, self.labels)
+
+        def iter_setup(self):
+            pass
+    
+    sequencial_pipe = SequencialOPPipeline(batch_size=batch_size, num_threads=1, device_id = 0, num_gpus = 1)
+    sequencial_pipe.build()
+    sequencial_pipe_out = sequencial_pipe.run()
+    fused_pipe = FusedOPPipeline(batch_size=batch_size,
+                          num_threads=1,
+                          device_id = 0, num_gpus = 1)
+    fused_pipe.build()
+    fused_pipe_out = fused_pipe.run()
+    for i in range(batch_size):
+        sequencial_pipe_out_cpu = sequencial_pipe_out[0].as_cpu()
+        fused_pipe_out_cpu = fused_pipe_out[0].as_cpu()
+        assert(np.sum(sequencial_pipe_out_cpu.at(i)-fused_pipe_out_cpu.at(i))==0)
```

Comment:
```suggestion
        assert(np.sum(np.abs(sequencial_pipe_out_cpu.at(i)-fused_pipe_out_cpu.at(i)))==0)
```
np.abs was necessary here